src/neuroconv/utils/json_schema.py

```diff
@@ -14,17 +14,25 @@
 
 
 class NWBMetaDataEncoder(json.JSONEncoder):
-    def default(self, o):
+    def default(self, obj):
```

Comment:
Opinion: `obj` slightly more descriptive than `o`. Can't use `object` as a name since that's a special Python built-in. It's also only used by `json` library as a positional argument so what we name it here shouldn't matter as long as it's easily readable